app/controllers/locations/ips_controller.rb

```diff
@@ -6,12 +6,13 @@ def new
   end
 
   def create
-    @ip = @location.ips.new(ip_params)
-    if @ip.save
+    if ips.all?(&:valid?)
+      ips.each{|ip| ip.save}
```

Comment:
You should be able to just save the params as an array of arrays, eg: the following works:
```
  Ip.create([[address: "1.1.1.1", location_id: 2], [address: "2.2.2.2", location_id: 2]])
```

Can you get it to pass in params like that?